CMakeLists.txt

```diff
@@ -108,6 +108,19 @@ if(HAIKU)
   set(UBSAN OFF)
 endif()
 
+if(AMIGA)
+  set(ASAN OFF)

```

Comment:
Consider moving this to `CMake/amiga_defs.cmake` like we do for other platforms.